src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -274,6 +274,18 @@
     type: command
     short-summary: Unregister a provider.
 """
+helps['provider operation'] = """
+    type: group
+    short-summary: get provider operations.
+"""
+helps['provider operation show'] = """
+    type: command
+    short-summary: Get an individual provider's operations.
+"""
+helps['provider operation list'] = """
+    type: command
+    short-summary: Get operations from all providers.
```

Comment:
I should update the help to get it clear.
`show` returns the type `ProviderOperationsMetadata` which has name, type and a list of operations allowed, etc.  `list` returns `ProviderOperationsMetadataPaged` which is an array of `ProviderOperationsMetadata`. It is very similar like the result from `vm show` or `vm list`
I am not convinced we should convert the result of `show` to a list, as this will incur the inconsistencies with other commands.